rules/import.json

```diff
@@ -4,6 +4,7 @@
         "import"
     ],
     "settings": {
+        "import/extensions": [".js", ".jsx"],
```

Comment:
`json` too?

Reply:
needed on resolve, but not here. this is just which files to parse for exports